demo/admin/src/products/ProductsGrid.tsx

```diff
@@ -191,7 +234,8 @@ export function ProductsGrid() {
             flex: 1,
             minWidth: 130,
             type: "boolean",
-            valueGetter: (params) => params.row.status == "Published",
+            disableExport: true, // boolean-value is not allowed for export, no way to provide a different value for excel-export
+            valueGetter: (params) => params.row.status == "Published", // why is this here?
```

Comment:
because the demo was ported from a boolean status to enum status (this should be done properly)